.github/dependabot.yml

```
@@ -0,0 +1,41 @@
+    reviewers:
+      - "@me"
+    assignees:
+      - "@me"
+    commit-message:
+      prefix: "deps"
+      include: "scope"
+    labels:
+      - "dependencies"
+      - "npm"
+    versioning-strategy: "increase"
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)